src/fosslight_reuse/cli.py

```diff
@@ -29,6 +30,9 @@ def main():
     if args.help:
         print_help_msg()
 
+    if args.version:
+        print_version(PKG_NAME)
+
```

Comment:
`sys.exit(0)` is needed.
or add this to below if statement.

Reply:
ex. https://github.com/fosslight/fosslight_scanner/blob/main/src/fosslight_scanner/cli.py
ex2. https://github.com/fosslight/fosslight_dependency_scanner/blob/main/src/fosslight_dependency/run_dependency_scanner.py